docs/contributing/pull-request-guidelines/index.md

```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. Select one based on the following condition.
```

Comment:
[style]
- I think `-` is unnecessary here.
- template -> templates?

```suggestion
There are two types of templates. Select one based on the following condition.
```

Reply:
Thank you!